Sources/App/Core/Extensions/S3Store+ext.swift

```diff
@@ -22,8 +22,8 @@ extension S3Store {
         guard let bucket = Current.awsReadmeBucket() else {
             throw Error.genericError("AWS_README_BUCKET not set")
         }
-        let key = Key(bucket: bucket, path: "\(owner)/\(repository)/readme.html")
-        guard let body = try await client.get(URI(string: key.url)).body else {
+        let url = "https://\(bucket).s3-us-east-2.amazonaws.com/\(owner)/\(repository)/readme.html"
+        guard let body = try await client.get(URI(string: url)).body else {
```

Comment:
We should push that url generation into `S3Store.Key`, using `S3Store.region`. I'll fix that up tomorrow and then merge it.